frontend/src/__tests__/components/widget/InlineTab/InlineTabWrapper.test.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import nock from 'nock';
+import { shallow, mount } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import merge from 'merge';
+import viewHandler from '../../../../reducers/viewHandler';
+import InlineTabWrapper from '../../../../components/widget/InlineTabWrapper';
+import hotkeys from '../../../../../test_setup/fixtures/hotkeys.json';
+import keymap from '../../../../../test_setup/fixtures/keymap.json';
+import { ShortcutProvider } from '../../../../components/keyshortcuts/ShortcutProvider';
+import { initialState as appHandlerState } from '../../../../reducers/appHandler';
+import { initialState as windowHandlerState } from '../../../../reducers/windowHandler';
+import tablesHandler from '../../../../reducers/tables';
+import { Provider } from 'react-redux';
+import props from '../../../../../test_setup/fixtures/widget/inlinetab/inline_tab_wrapper.json';
+import tabData from '../../../../../test_setup/fixtures/widget/inlinetab/inline_tab_data.json';
+import inlineTabStoreMore from '../../../../../test_setup/fixtures/widget/inlinetab/inline_tab_data_more.json';
+import inlineTabStore from '../../../../../test_setup/fixtures/widget/inlinetab/inlineTabStore.json';
+import addNewData from '../../../../../test_setup/fixtures/widget/inlinetab/addNewData.json';
+import inlineTabInvalid from '../../../../../test_setup/fixtures/widget/inlinetab/inline_tab_invalid.json';
+import thunk from 'redux-thunk';
+const middlewares = [thunk];
+
+const mockStore = configureStore(middlewares);
+
+const createStore = function(state = {}) {
+  const res = merge.recursive(
+    true,
+    {
+      appHandler: {
+        ...appHandlerState,
+        me: { timeZone: 'America/Los_Angeles' },
+      },
+      windowHandler: { ...windowHandlerState },
+      ...viewHandler,
+      tables: { ...tablesHandler(undefined, {}) },
+    },
+    state
+  );
+
+  return res;
+};
+
+describe('InlineTabWrapper component', () => {
+  describe('rendering tests:', () => {
+    it('renders without errors', () => {
+      shallow(<InlineTabWrapper {...props} />);
+    });
+
+    it('renders a line properly', () => {
+      const initialState = createStore({
+        windowHandler: {
+          allowShortcut: true,
+          modal: {
+            visible: true,
+          },
+          inlineTab: inlineTabStore,
+        },
+      });
+      const store = mockStore(initialState);
+
+      nock(config.API_URL)
+        .defaultReplyHeaders({ 'access-control-allow-origin': '*' })
+        .get(`/window/123/2156425/AD_Tab-222/`)
```

Comment:
![Screenshot 2020-12-17 at 09 24 24](https://user-images.githubusercontent.com/1708561/102456361-ae89f880-4049-11eb-9690-95a3a5eef729.png)
If I am changing that with the values from the fixtures for some reason nock is giving these errors. So having bunch of errors would fill up the logs. It's a test, I am going to leave it hardcoded for now as I don't want to spend more time investigating why nock is trowing those errors in case I am using the ids from the fixture